deployment/terraform/clear_opensearch/clear_opensearch.sh.tpl

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo -e "\nDelete OpenSearch index\n"
```

Comment:
Let's make it plural — **indexes**. In the near future, we will have a few more indexes, so it won't just be one.